inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/InAppMessageBaseView.kt

```diff
@@ -118,18 +115,9 @@ internal open class InAppMessageBaseView(context: Context, attrs: AttributeSet?)
     @SuppressLint("ClickableViewAccessibility")
     private fun bindImage() { // Display image.
         if (!this.imageUrl.isNullOrEmpty()) {
-            val draweeView = message_image_view
-            if (draweeView != null) {
-                draweeView.setOnTouchListener(this.listener)
-                // Building a DraweeController to handle animations.
-                // Image should be already downloaded and cached in memory. Fresco library will look for the
-                // cached image by URI.
-                draweeView.controller = Fresco.newDraweeControllerBuilder()
-                        .setUri(this.imageUrl)
-                        .setAutoPlayAnimations(true)
-                        .build()
-                draweeView.aspectRatio = this.imageAspectRatio
-                draweeView.visibility = View.VISIBLE
+            if (message_image_view != null) {
+                message_image_view.setOnTouchListener(this.listener)
```

Comment:
Until this is called, when campaign view's alpha is 0, user is able to interact with the app?

Reply:
No, user will not be able to interact since modal and fullscreen types cover the whole screen.

Reply:
So the UI is blocked until image is loaded?

Reply:
Yes